kplutus-pyk/src/kplutus_pyk/genv.py

```diff
@@ -0,0 +1,91 @@
+from typing import Any, List, Tuple
+
+from pyk.kast import (
+    KApply,
+    KAtt,
+    KFlatModule,
+    KImport,
+    KInner,
+    KLabel,
+    KProduction,
+    KRewrite,
+    KRule,
+    KTerminal,
+    KToken,
+    KVariable,
+)
+from pyk.ktool import KRun
+
+_lam = '(lam__)_UPLC-SYNTAX_Term_UplcId_Term'
+_app = '[__]_UPLC-SYNTAX_Term_Term_TermList'
```

Comment:
```suggestion
_LAM: Final = '(lam__)_UPLC-SYNTAX_Term_UplcId_Term'
_APP: Final = '[__]_UPLC-SYNTAX_Term_Term_TermList'
```

(`from typing import Final`)